app/src/mainwindow2.h

```diff
@@ -65,6 +66,10 @@ class MainWindow2 : public QMainWindow
     void undoActSetEnabled();
     void updateSaveState();
     void clearRecentFilesList();
+    void pegBarReg();
```

Comment:
The intent of the function could be clearer, we're launching a dialog, so the method could be called openPegBarRegDialog or pegBarDialogPressed or similar.